app/views/users/show.html.erb

```diff
@@ -1 +1,25 @@
-<h1><%= @user.name %></h1>
\ No newline at end of file
+<div class="show_container">
+<img class="pic" src="<%= @user.photo %>" alt="profile_pic">
+<div class= "name">
+<%= @user.name%>
+<p class="post-count"><%= @user.posts_counter%>
+</div>
+</div>
+<div class="bio">
+<%= @user.bio%>
+</div>
+<div class="posts">
+<% @user.posts.each do |d|%>
+    <div class="post">
+    <p class="title"> <%= d.title %></p>
+    <p> <%= d.text %></p>
+    <div class="counts">
+    <p class="likes-count"> <%= d.likes_counter%></p>, 
+    <p class="comments-count"> <%= d.comments_counter%></p>
+    </div>
+  </div>
+<%end%>
+</div>
+<button class="see_all">
+<%= link_to "see all posts", user_posts_path(@user)%>
+</button>
```

Comment:
- [ ] @JacintaNgwu great work implementing the changes requested, however, you forgot to add the post counter title before the counter, the same case for the likes and comments counters 
![image](https://user-images.githubusercontent.com/61408860/233772895-25810e1b-5694-4706-9446-15a4ed93ea92.png)